tutorials/3d/using_voxel_gi.rst

```diff
@@ -0,0 +1,209 @@
+  is changed in any way after baking, the VoxelGI node must be baked again.
+  Otherwise, indirect lighting will look incorrect.
+- **Dynamic:** The mesh won't be taken into account for VoxelGI baking, but it will
+  still receive *and* contribute indirect lighting to the scene in real-time.
+  This option is much slower compared to **Static**. Only use the **Dynamic**
+  global illumination mode on large meshes that will change significantly during gameplay.
+
+Additionally, there are 3 bake modes available for lights
+(DirectionalLight3D, OmniLight3D and SpotLight3D):
+
+- **Disabled:** The light won't be taken into account for VoxelGI baking.
+  The light won't contribute indirect lighting to the scene.
+- **Static:** The light will be taken into account for VoxelGI baking.
+  The light will contribute indirect lighting to the scene. If the light
+  is changed in any way after baking, the VoxelGI node must be baked again or
+  indirect lighting will look incorrect. If in doubt, use this mode for level lighting.
+- **Dynamic (default):** The light won't be taken into account for VoxelGI baking,
+  but it will still contribute indirect lighting to the scene in real-time.
+  This option is slower compared to **Static**. Only use the **Dynamic** global
+  illumination mode on lights that will change significantly during gameplay.
+
+.. note::
+
+    The amount of indirect energy emitted by a light depends on its color,
+    energy *and* indirect energy properties. To make a specific light emit more
+    or less indirect energy without affecting the amount of direct light emitted
+    by the light, adjust the **Indirect Energy** property in the Light3D inspector.
+
+Which global illumination mode should I use?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is no universally "better" global illumination mode to use,
+but here are some recommendations for meshes:
+
+- For static level geometry, use the **Static** global illumination mode *(default)*.
+- For small dynamic geometry and players/enemies, use the **Disabled** global
+  illumination mode. Small dynamic geometry will not be able to contribute a significant
+  amount of indirect lighting, due to the geometry being smaller than a voxel.
+  If you need indirect lighting for small dynamic objects, it can be simulated
+  using an OmniLight3D or SpotLight3D node parented to the object.
+- For *large* dynamic level geometry (such as a moving train),
+  use the **Dynamic** global illumination mode.
+
+Here are some recommendations for light bake modes:
+
+- For static level lighting, use the **Static** bake mode.
+  The **Static** mode is also suitable for dynamic lights that don't change
+  much during gameplay, such as a flickering torch.
+- For short-lived dynamic effects (such as a weapon), use the **Disabled**
+  bake mode to improve performance.
+- For long-lived dynamic effects (such as a rotating alarm light),
+  use the **Dynamic** bake mode to improve quality *(default)*.
+
```

Comment:
Since this section is in both the VoxelGI documentation and the SDFGI docs, its probably better suited to be moved to the global illumination docs

Reply:
I probably left this by mistake – I'll move it to the Introduction to global illumination page.